Makefile

```diff
@@ -133,14 +133,15 @@ manifests: $(CONTROLLER_GEN) #EXHELP Generate WebhookConfiguration, ClusterRole,
 	mv $(CRD_WORKING_DIR)/olm.operatorframework.io_clustercatalogs.yaml $(KUSTOMIZE_CATD_CRDS_DIR)
 	rmdir $(CRD_WORKING_DIR)
 	# Generate the remaining operator-controller manifests
-	$(CONTROLLER_GEN) rbac:roleName=manager-role paths="./internal/operator-controller/..." output:rbac:artifacts:config=$(KUSTOMIZE_OPCON_RBAC_DIR)
+	$(CONTROLLER_GEN) rbac:roleName=manager-role paths="./internal/operator-controller/controllers/..." output:rbac:artifacts:config=$(KUSTOMIZE_OPCON_RBAC_DIR)
```

Comment:
The problematic dep is called in the controller :-( 
I thought in extra in a utils and then do something like that
it might works

Reply:
That might just be my crazy idea I mentioned earlier